src/main/java/com/galvanize/customer/controllers/CustomerRestController.java

```diff
@@ -1,7 +1,40 @@
 package com.galvanize.customer.controllers;
 
-import org.springframework.web.bind.annotation.RestController;
+import com.galvanize.customer.entities.Customer;
+import com.galvanize.customer.services.CustomerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
 
 @RestController
 public class CustomerRestController {
+
+    @Autowired
+    CustomerService service;
+
+    @PostMapping("/add")
+    public Customer addCustomer(@RequestBody Customer c){
+        return service.addCustomer(c);
+    }
+
+    @GetMapping("/all/{state}")
```

Comment:
State should be in a query string (@RequestParam). Something like /all?state=DC. That way, the same endpoint can be multifunctional.  Save @PathVariable for identifiers, like ids